fastlane/swift/LaneFileProtocol.swift

```diff
@@ -50,6 +50,10 @@ public class LaneFile: NSObject, LaneFileProtocol {
         return String(laneName.prefix(laneName.count - 12))
     }
 
+	private static func trimLaneWithoutOptionsFromName(laneName: String) -> String {
+		return String(laneName.prefix(laneName.count - 16))
```

Comment:
If we remove the `:` from the comment below, this should be changed to `15`

Reply:
As mentioned above:
A lane with parameters for example ```func thirdLane(withOptions options:[String: String]?) { ... }``` is going to have the following signature: ```thirdLaneWithOptions:```, that's why removing 4 characters, it was left ```thirdLaneWithOpti```